scripts/inline_comments_data_collection.py

```diff
@@ -0,0 +1,154 @@
+import json
+import logging
+import os
+
+from libmozdata.phabricator import PhabricatorAPI
+
+from bugbug.tools.code_review import PhabricatorReviewData
+from bugbug.utils import get_secret
+
+review_data = PhabricatorReviewData()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+os.makedirs("patches", exist_ok=True)
+os.makedirs("comments", exist_ok=True)
+os.makedirs("dataset", exist_ok=True)
+
+
+class PhabricatorClient:
+    def __init__(self):
+        self.api = PhabricatorAPI(get_secret("PHABRICATOR_TOKEN"))
+
+    def find_revision_from_patch(self, patch_id):
+        diffs = self.api.search_diffs(diff_id=patch_id)
+
+        if not diffs:
+            raise Exception(f"No diffs found for the given patch ID: {patch_id}")
```

Comment:
It would be better to create a custom error type instead of using `Exception` directly: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/broad-exception-raised.html

Reply:
Done here 5d9b655222bae7acbdc0d53fdf20f7a0cd52f56d